.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,44 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+	"name": "C# (.NET)",
```

Comment:
I think the standard in this repo is 4 spaces. can you please remove tabs?

Reply:
Maybe it's not, but as far as I know this repo is 4 spaces everywhere - all the core dotnet repos are.

Reply:
I just did check the runtimes repo [file](https://github.com/dotnet/runtime/blob/main/.devcontainer/devcontainer.json) it uses `tabs`. 
I already changed to spaces, but don't know if I should enable the 4 size indent_size. (default is 2)

I would change it like this:
![image](https://github.com/dotnet/aspire/assets/11291885/e2fefe46-3503-44e5-8af9-49d6fa3fd599)

But that means that every file will get the 4 space wide indent.

Reply:
Ah - ok fine to leave consistent with runtime and we can leave settings as they are for now.